Java/AutoML/automl-cms-api/src/main/java/com/automl/cms/api/handler/MLExceptionHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.automl.cms.api.handler;
+
+import java.util.Collections;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import com.automl.component.constants.MLException;
+import com.automl.component.constants.ResultCode;
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * global exception Handler
+ *
+ * @author wangqingwei
+ * Created on 2022-08-12
+ */
+@ControllerAdvice
+public class MLExceptionHandler {
+
+    @ResponseBody
+    @ExceptionHandler({MLException.class})
+    public Object exceptionHandler(MLException exception) {
+        return ImmutableMap.of("error", ImmutableMap.of(
+            "code", exception.getCode().getValue(),
+            "message", exception.getMsg(),
```

Comment:
Why not use getMessage like the following standard Exception?

Reply:
ok i mv msg to message